### PR TITLE
Mobile 3915

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

File: src/core/features/course/services/course-helper.ts [hunk 1]
```diff
@@ -2020,20 +2020,25 @@ export class CoreCourseHelperProvider {
...
      * @param course Course to open
-     * @param params Params to pass to the course page.
-     * @param siteId Site ID. If not defined, current site.
+     * @param navOptions Navigation options that includes params to pass to the page.
      * @return Promise resolved when done.
      */
-    async openCourse(course: CoreCourseAnyCourseData | { id: number }, params?: Params, siteId?: string): Promise<void> {
+    async openCourse(
+        course: CoreCourseAnyCourseData | { id: number },
+        navOptions?: CoreNavigationOptions & { siteId?: string },
```
Comment (dpalou) [inline src/core/features/course/services/course-helper.ts:2028]: Please document the parameters change in upgrade.txt.

**Review by dpalou — CHANGES_REQUESTED** [review]

File: src/core/features/course/services/course-helper.ts (hunk 1 above)
Comment (crazyserver) [reply src/core/features/course/services/course-helper.ts:2028]: I knew I was missing something! Thanks!

File: src/core/features/course/lang.json [hunk 2]
```diff
@@ -49,7 +50,6 @@
     "overriddennotice": "Your final grade from this activity was manually adjusted.",
     "refreshcourse": "Refresh course",
     "section": "Section",
-    "sections": "Sections",
```
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/features/course/lang.json:52]: Let's chat about this!

[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/features/course/lang.json:52]: It won delete unnecessary strings!

**Review by dpalou — APPROVED** [review]

**Pull request merged**